k8s/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: datadelivery-kafka-rest-proxy
+  name: datadelivery-kafka-rest-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: datadelivery-kafka-rest-proxy
+  template:
+    metadata:
+      labels:
+        app: datadelivery-kafka-rest-proxy
+    spec:
+      containers:
+        - image: confluentinc/cp-kafka-rest:6.2.0 # TODO: Paremetrize tag
```

Comment:
Couldn't you just parametrize the tag and add it to the variable groups?

Reply:
I could. That's a remnant from the schema registry deployment so might as well get that done now...